.editorconfig

```diff
@@ -134,7 +134,7 @@ csharp_preserve_single_line_statements = true
 csharp_preserve_single_line_blocks = true
 csharp_using_directive_placement = outside_namespace:silent
 csharp_prefer_simple_using_statement = true:suggestion
-csharp_style_namespace_declarations = block_scoped:silent
+csharp_style_namespace_declarations = file_scoped:silent
```

Comment:
Changed .editorconfig here so new files are created with file-scoped namespace, but it doesn't mind that old files might stay with block-scoped namespace for a while.